float64 timestamp

float64 robotLocX
float64 robotLocY
float64 robotAngle

float64 windowSize

float32[] lines_p1x
float32[] lines_p1y
float32[] lines_p2x
float32[] lines_p2y
uint32[]   lines_col

float32[] points_x
float32[] points_y
uint32[]   points_col

float32[] circles_x
float32[] circles_y
uint32[]   circles_col
